packages/editor-ui/src/components/SettingsSidebar.vue

```diff
@@ -149,36 +158,15 @@ export default mixins(
 	}
 }
 
+.usersMenu svg:global(.svg-inline--fa) { left: -2px; }
+.apiMenu svg:global(.svg-inline--fa) { left: 2px; }
```

Comment:
could these be global changes? could we update global theme or else put this behind a prop?

Reply:
Those area fine tunnings that are specific to each icon to keep them aligned so I am not sure they should be a prop. But I could get rid of the global selectors perhaps by targeting and `svg` elements.

Reply:
ideally we should not have to do these tweaks.. I mean what if an embeder wants to replace all our icons.. our components should work without this

Reply:
but that's only in an ideal world 😊

Reply:
Agree 100%. [Here](https://n8n-io.github.io/n8n-storybook/?path=/story/atoms-menu--primary), for example, everything is aligned without any tweaks. So really looking forward to getting it done and merged.